pp/serv/rpc.go

```diff
@@ -25,10 +25,9 @@ const (
 	FILE_DATA_SAFE_SIZE = 3500000
 
 	// timeout for waiting result from external source, in seconds
-	WAIT_TIMEOUT time.Duration = 3
+	WAIT_TIMEOUT time.Duration = 5
```

Comment:
why not directly use time.Second or time.MilliSecond

Reply:
Good point. Changing it.